tests/src/python/test_selective_masking.py

```diff
@@ -146,6 +147,7 @@ def tearDownClass(cls):
         if (os.environ.get('QGIS_CONTINUOUS_INTEGRATION_RUN', None) != "true" and
                 cls.report != REPORT_TITLE):
             QDesktopServices.openUrl(QUrl(f"file:///{report_file_path}"))
```

Comment:
You can now remove this part and the cls.report variable and initialization

Reply:
I'd like to defer this to a later pr -- we need to go through one by one and remove any custom render check methods implemented in each test file. But that will require moving whatever extra custom logic those checks have up to the base class methods wherever appropriate.... Basically a lot of work for each test that I'd rather is done separately (or as needed?).